zjit/src/hir.rs

```diff
@@ -439,6 +439,7 @@ pub enum SideExitReason {
     CalleeSideExit,
     ObjToStringFallback,
     UnknownSpecialVariable(u64),
+    TemporaryBug,
```

Comment:
I get that you want to leave the bug only temporarily, but it's always nice to describe what it is. Could we call it `UnknownDefinedType(op)` (also I feel like they are not "Unknown" to us but just "Unhandled" btw) or `DefinedMethod`?